src/plugins/oSnap/components/TransactionBuilder/TransferFunds.vue

```diff
@@ -85,18 +85,18 @@ watch(selectedTokenAddress, updateTransaction);
       <span class="text-skin-text">{{ $t('safeSnap.asset') }}</span>
       <AvatarToken
         :address="
-          selectedToken.address === 'main'
+          selectedToken?.address === 'main'
             ? ETH_CONTRACT
-            : selectedToken.address
+            : selectedToken?.address
         "
         class="ml-2"
       />
-      <span v-if="selectedToken">{{ selectedToken.symbol }}</span>
+      <span v-if="selectedToken">{{ selectedToken?.symbol }}</span>
       <span>
         {{
-          selectedToken.address === 'main'
+          selectedToken?.address === 'main'
             ? ''
-            : `(${shorten(selectedToken.address)})`
+            : `(${shorten(selectedToken?.address)})`
```

Comment:
had to make this optional incase things were undefined

Reply:
If `selectedToken` or `address` is undefined, it may print `'(undefined)'` right? (I didn't test)